atom/browser/api/atom_api_global_shortcut.cc

```diff
@@ -43,14 +45,26 @@ bool GlobalShortcut::RegisterAll(
     const std::vector<ui::Accelerator>& accelerators,
     const base::Closure& callback) {
   std::vector<ui::Accelerator> registered;
+  std::vector<std::string> mediaKeys = {"Media Play/Pause", "Media Next Track",
+                                        "Media Previous Track"};
+
   for (auto& accelerator : accelerators) {
+#if defined(OS_MACOSX)
+    std::string shortcutText = base::UTF16ToUTF8(accelerator.GetShortcutText());
+    if (std::find(mediaKeys.begin(), mediaKeys.end(), shortcutText) !=
```

Comment:
can we check the actual keys instead of the text?

Reply:
the only accessor chromium gives us is `GetShortcutText()`; i wanted to check the keys originally but i'm not sure it's possible 🤔